strongbox-web-core/src/main/java/org/carlspring/strongbox/controllers/configuration/security/authorization/AuthorizationConfigController.java

```diff
@@ -79,7 +90,7 @@
                  produces = { MediaType.TEXT_PLAIN_VALUE, MediaType.APPLICATION_JSON_VALUE })
     public ResponseEntity addRole(@RequestBody @Validated RoleForm roleForm,
                                   BindingResult bindingResult,
-                                  @RequestHeader(HttpHeaders.ACCEPT) String acceptHeader)
+                                  @RequestHeader(HttpHeaders.ACCEPT) String acceptHeader) throws IOException
```

Comment:
Shouldn't we add endpoint to update custom roles ?

Reply:
Perhaps we should...?

@sbespalov , @steve-todorov : Any thoughts on this?

Perhaps @ptirador might like to join in and look into this as a follow-up task...? :)

Reply:
> Perhaps @ptirador might like to join in and look into this as a follow-up task...? :)

yes, I mentioned this in PR description, that there should be such follow-up task
but first @steve-todorov  should provide requirements of what API needed